src/java.base/share/classes/jdk/internal/util/ClassFileDumper.java

```diff
@@ -80,7 +79,7 @@ public static ClassFileDumper getInstance(String key, String path) {
     private final AtomicInteger counter = new AtomicInteger();
 
     private ClassFileDumper(String key, String path) {
-        String value = GetPropertyAction.privilegedGetProperty(key);
+        String value = VM.getSavedProperty(key);
```

Comment:
Could add a comment that this was necessary to avoid a bootstrap circularity issue in the java/lang/String/concat/WithSecurityManager.java test